src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -38,11 +38,23 @@ trait GoogleServicesDAO {
 
   def isUserInProxyGroup(user: RawlsUser): Future[Boolean]
 
```

Comment:
instead of passing in groupName to the functions below pass RawlsGroup or RawlsGroupRef. Insead of memberEmail, RawlsUser. Maybe add/remove methods for sub groups unless you want the api to take Either[RawlsUser, RawlsGroup]. In the case of user, need to proxyify.